extensions/virtual-threads/runtime/src/main/java/io/quarkus/virtual/threads/VirtualThreadsConfig.java

```diff
@@ -0,0 +1,32 @@
+package io.quarkus.virtual.threads;
+
+import java.time.Duration;
+import java.util.Optional;
+
+import io.quarkus.runtime.annotations.ConfigItem;
+import io.quarkus.runtime.annotations.ConfigPhase;
+import io.quarkus.runtime.annotations.ConfigRoot;
+
+@ConfigRoot(phase = ConfigPhase.RUN_TIME)
+public class VirtualThreadsConfig {
+
+    /**
+     * Virtual thread name prefix. If left blank virtual threads will be unnamed.
+     */
+    @ConfigItem(defaultValue = "quarkus-virtual-thread-")
+    Optional<String> namePrefix;
+
+    /**
+     * The shutdown timeout. If all pending work has not been completed by this time
+     * then any pending tasks will be interrupted, and the shutdown process will continue
+     */
+    @ConfigItem(defaultValue = "1M")
```

Comment:
that's a lot... I think for the rest we use 10s.

Reply:
I got the same timeout as `thread-pool.shutdown-Timeout`. But yeah it sure seems like a lot.

Reply:
Hum, didn't know that. For Kafka didn't we set it to 10s?

Reply:
Yes for kafka producers it is 10s. For consumers it was twice the poll timeout IIRC